pages/index.js

```diff
@@ -1,14 +1,18 @@
 import styled from "styled-components";
+import { useState } from "react";
 
 import ShopHeader from "@/components/ShopHeader";
 import Gallery from "@/components/Gallery";
 
 export default function HomePage() {
+  const [cartTotal, setCartTotal] = useState(0);
+
   return (
     <AppContainer>
-      <ShopHeader />
+      <ShopHeader cartTotal={cartTotal} />
       <main>
-        <Gallery />
+        <Gallery setCartTotal={setCartTotal} cartTotal={cartTotal} />
```

Comment:
Please never pass the set-function to other components. Always keep them in the file where you initialize them. 
I'm not sure about the following but it seems like you pass the cartTotal and you never use it, so you don't need to pass it. 

```suggestion
        <Gallery onSetCartTotal={handleCartTotal}  />
```

Reply:
- [x] Replaced pass through of setter with handle function